### PR TITLE
add test case

### PR DESCRIPTION
**Pull request by 15596858998**
add test case

### CODE REVIEW

File: tests/playTests.sh [hunk 1]
```diff
@@ -184,10 +184,16 @@ fi
 
 
 println "\n===>  simple tests "
-
+datagen
```
Comment (terrelln) [inline tests/playTests.sh]: Why did you add this call to `datagen`? This will output a ton of garbage to `stdout`.

File: tests/playTests.sh [hunk 2]
```diff
@@ -785,6 +792,7 @@ println "world!" > world.tmp
 cat hello.tmp world.tmp > helloworld.tmp
 zstd -c hello.tmp > hello.zst
 zstd -c world.tmp > world.zst
+zstd -c hello.tmp world.tmp > helloworld.zst
```
Comment (terrelln) [inline tests/playTests.sh:795]: In order for this to be a useful test, you would want to decompress it and diff it. E.g. add the lines:

```
zstd -dc helloworld.zst > result.tmp
$DIFF helloworld.tmp result.tmp
```

File: tests/playTests.sh [hunk 3]
```diff
@@ -1549,6 +1565,7 @@ if [ -n "$hasMT" ]
 then
     println "\n===>   adaptive mode "
     roundTripTest -g270000000 " --adapt"
+    roundTripTest -g270000000 " --adapt=" && die "invalid compression parameter"
```
Comment (terrelln) [inline tests/playTests.sh]: This shouldn't be a `roundTripTest`.

Try moving this down after line 1573, and doing:

```
zstd --adapt= tmp && die "invalid compression parameter"
```

**Review by terrelln — CHANGES_REQUESTED** [review]

File: tests/playTests.sh (hunk 1 above)
Comment (15596858998) [reply tests/playTests.sh]: To test whether the function of the Datagen without parameters by default is normal.

File: tests/playTests.sh (hunk 2 above)
Comment (15596858998) [reply tests/playTests.sh:795]: ok，I'll try it.

File: tests/playTests.sh (hunk 3 above)
Comment (15596858998) [reply tests/playTests.sh]: It's like this.

File: tests/playTests.sh (hunk 1 above)
Comment (felixhandte) [reply tests/playTests.sh]: That's already tested by the following line.

**Comment by 15596858998** [comment]
@terrelln

**Review by terrelln — APPROVED** [review]
